### PR TITLE
Disable System.Reflection.Metadata 1.2

### PR DESCRIPTION
**Pull request by ericstj**
Remove the 'future' configuration of System.Reflection.Metadata and
build the live assembly as 1.3 with future defined.

The 1.2 build is no longer needed in the dev branch as this will be
shipped stable from Rel.

** Note: do not integrate this to rel until after we have shipped 1.2 **

/cc @tmat @nguerrera @weshaggard

### CODE REVIEW

**Comment by nguerrera** [comment]
:+1:

File: src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj [hunk 1]
```diff
@@ -17,8 +11,9 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <NoWarn>1591</NoWarn>
     <CLSCompliant>false</CLSCompliant>
-    <AssemblyVersion Condition="'$(Future)' != 'true'">1.2.0.0</AssemblyVersion>
-    <DefineConstants>$(DefineConstants);SRM</DefineConstants>
+    <AssemblyVersion>1.3.0.0</AssemblyVersion>
+    <!-- TODO: Remove use of FUTURE -->
```
Comment (weshaggard) [inline src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj]: @tmat @nguerrera is this TODO something one of you guys can pick-up?

Comment (ericstj) [inline src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj]: @tmat mentioned he'd do it.

Comment (tmat) [inline src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj]: Yes, I'll do it.

**Pull request merged**